src/main/resources/lang/installer.properties

```diff
@@ -31,6 +31,6 @@ prompt.server.jar.invalid=No valid {0} server jar found
 prompt.server.generate=Generate
 prompt.server.overwrite=Are you sure you want to override the existing launch scripts?
 prompt.install.successful.title=Successfully Installed
-prompt.install.successful=Fabric Loader {0} for {1} has been successfully installed.<br>Many mods also require you to download <a href="{2}">Fabric API</a> mod into the mods folder.
+prompt.install.successful=Fabric Loader {0} for {1} has been successfully installed.<br>Many mods also require you to put <a href="{2}">Fabric API</a> into the mods folder.
```

Comment:
Imo mentioning mods folder is a bad idea. Users are supposed to know how to install mods; otherwise there is no point for them to install the api via this installer.

Reply:
Imo we might link to how to install a mod on the wiki, i.e. https://fabricmc.net/wiki/tutorial:adding_mods

Reply:
IMO we shouldn't care about users that are using modpacks, this installer is only for the vanilla launcher, in which there are no easy way to install a modpack, people who are using the vanilla launcher to install a modpack should know what they are doing since better options are usually offered such as multimc and twitch.

People who are installing via this installer are mostly the people that are installing mods themselves, which it is not bad to mention about the mods folder.

The wiki page sounds nice but I don't know how much that can help, you should be able to install mods into the mods folder as a user.

Also that wiki page is full of text and doesn't really differentiate itself from just "please add to mods folder"

Reply:
This installer is only used for installing into the mods folder. If automated methods are using this then something is prossibly wrong. (the meta api makes it trivial to re-impliment the installer).

Linking to a wiki page might not be a bad idea, or improving the curse page. (If someone wants to help with this let me know as writing things isnt my kind of thing)

Reply:
Imo we should keep the term "installing", and add a `<a href="https://fabricmc.net/wiki/tutorial:adding_mods">installing</a>` so people who don't know how to install a mod can go to that link and learn that putting the mod jar in the right folder is the way to go.

Reply:
For the record, both https://fabricmc.net/wiki/tutorial:install_with_minecraft_launcher and https://fabricmc.net/wiki/tutorial:install_with_multimc explain how to install the Fabric API and also show how to install mods, using modmenu as an example.

Reply:
It clearly isn't enough

Reply:
In response to @liach 's initial:
Yeah, I understand your point. It is entirely reasonable to expect the user to know how to install mos. At the same time, I've had to deal with users (at multiple points) who have required step-by-step instructions on how to locate their minecraft installation, including a note that it is not on their desktop, and that they should not try to execute the minelittlepony-mod jar file.

So I think the first step is to assume nothing. The user is a brick.

Reply:
Also would it help if at the end the installer had a button that just opens the mods folder for them?

Reply:
Both as a convenience, and an aid to the bricks out there.

Reply:
It is not viable to show the mods folder unless if create profile is checked, since this installer is just installing fabric loader into launcher versions.